Yona/Yona/DataService/APIServiceManager.swift

```diff
@@ -539,20 +539,6 @@ extension APIServiceManager {
                             return
                         }
                         self.newUser = Users.init(userData: json)
-                        //intialise the goals and the activities
-                        self.getActivitiesArray({ (success, message, code, activities, error) in
-                            if success {
-                                self.getAllTheGoalsArray({ (success, message, code, goals, error) in
-                                    if success {
-                                        onCompletion(true, self.serverMessage, self.serverCode,self.newUser)
-                                    } else {
-                                        onCompletion(false, self.serverMessage, self.serverCode, self.newUser)
-                                    }
-                                })
-                            } else {
-                                onCompletion(false, self.serverMessage, self.serverCode, self.newUser)
-                            }
-                        })
```

Comment:
I think you remove completion block here.
onCompletion(true, self.serverMessage, self.serverCode,self.newUser)